lib/app.js

```diff
@@ -72,5 +72,43 @@ app.get('/api/todos', async (req, res) => {
   }
 });
 
+app.put('/api/todos/:id', async (req, res) => {
+  try {
+    const todo = req.body;
+    console.log(todo);
+    const data = await client.query(`
+      UPDATE todos
+      SET task = $1, completed = $2
+      WHERE user_id = $3
+      AND id = $4
+      RETURNING id, task, completed, user_id as "userId";
+      `,
+    [todo.task, todo.completed, req.params.id, req.userId]);
```

Comment:
switch `req.params.id` and `req.userId` to match the order `user_id = $3` and `id = $4`